docs/angular-testing-library/api.mdx

```diff
@@ -411,8 +411,8 @@ expect(screen.getByTestId('name-value').textContent).toBe('Sarah')
 
 ### `rerender`
 
-Re-render the same component with different props. Input properties that are not
-defined are cleared. This calls `detectChanges` after the props are updated.
+Create and render a new instance of the component.
```

Comment:
I would like to keep a note about "Input properties that are not
defined are cleared.".

Now that I see this, this is also the reason that `change` and `rerender` exist.
With change you can't clear input properties.

https://github.com/testing-library/angular-testing-library/issues/252

Reply:
I updated the PR. Please let me know what you think. I also created https://github.com/testing-library/angular-testing-library/issues/365 for further discussion. Thanks again for your great fast responses :)